tests/test.py

```diff
@@ -964,12 +964,12 @@ def test_rule_expressions_rtf(self):
                }
 
         sample = factory.make_sample(tests_data_dir+'/office/blank.doc', metainfo=part)
-        rule = ExpressionRule(CreatingConfigParser(config))
+        rule = ExpressionRule(CreatingConfigParser(config), None)
```

Comment:
Did a previous change forget to add those or was there another change which makes them necessary now? How come this didn't blow up before?

Reply:
Addressed in #143 by squashing into the introducing commit.